discovery-provider/src/queries/get_feed.py

```diff
@@ -17,18 +17,9 @@ def get_feed(args):
     feed_results = []
     db = get_db_read_replica()
 
-    # filter should be one of ["all", "reposts", "original"]
-    # empty filter value results in "all"
-    if "filter" in args and args.get("filter") in ["all", "repost", "original"]:
```

Comment:
@piazzatron I wasn't sure if this was something that should remain in here or be pulled out. Thoughts?

Reply:
I like that you pulled it down, IMO this counts as param validation (enforcing a valid "enum" case), so it makes sense to be in the routing layer.